Clustering/Hdbscan/src/main/java/org/tribuo/clustering/hdbscan/HdbscanTrainer.java

```diff
@@ -740,29 +743,28 @@ private static DenseVector calculateOutlierScores(double[] pointNoiseLevels, int
                     numExemplarsThisCluster = outlierScoreIndexTree.size();
                 }
 
-                List<ClusterExemplar> subsetClusterExemplars = new ArrayList<>();
-
-                for (int i = 0; i < numExemplarsThisCluster; i++) {
-                    // Note that for non-outliers, the first node is polled from the tree, which has the lowest outlier
-                    // score out of the remaining points assigned this cluster.
-                    Entry<Double, Integer> entry = outlierScoreIndexTree.pollFirstEntry();
-                    subsetClusterExemplars.add(new ClusterExemplar(clusterLabel, entry.getKey(), data[entry.getValue()]));
-                }
-
-                // For each of the exemplars in this cluster, iterate the remaining nodes in the tree to find the maximum
-                // distance between the exemplar and the members of the cluster. The other exemplars don't need to be
-                // checked here since they won't be on the fringe of the cluster.
-                for (ClusterExemplar clusterExemplar : subsetClusterExemplars) {
+                // First, get the entries that will be used for cluster exemplars.
+                // Note that for non-outliers, the first node is polled from the tree, which has the lowest outlier
```

Comment:
I think this note is wrong now, as there are no outlier exemplars anymore, so it should just be "The first node is polled from the tree...".

Reply:
Yes, all the exemplars are now non-outliers, so this was ambiguous. It has been removed.